Source/BugsnagStackframe.m

```diff
@@ -7,7 +7,60 @@
 //
 
 #import "BugsnagStackframe.h"
+#import "BugsnagKeys.h"
+#import "BugsnagCollections.h"
 
 @implementation BugsnagStackframe
 
++ (NSDictionary *_Nullable)findImageAddr:(unsigned long)addr inImages:(NSArray *)images {
+    for (NSDictionary *image in images) {
+        if ([(NSNumber *)image[@"image_addr"] unsignedLongValue] == addr) {
+            return image;
+        }
+    }
+    return nil;
+}
+
++ (BugsnagStackframe *)frameFromDict:(NSDictionary *)dict
+                          withImages:(NSArray *)binaryImages {
+    BugsnagStackframe *frame = [BugsnagStackframe new];
+    frame.frameAddress = [dict[@"instruction_addr"] unsignedLongValue];
+    frame.symbolAddress = [dict[@"symbol_addr"] unsignedLongValue];
+    frame.machoLoadAddress = [dict[@"object_addr"] unsignedLongValue];
+    frame.machoFile = dict[@"object_name"];
+    frame.method = dict[@"symbol_name"];
```

Comment:
With these, is there any reason not to use the `KSCrashReportFields` constants?

Reply:
No reason - I've amended the implementation to use constants instead.